server/commitlog/commitlog.go

```diff
@@ -22,6 +22,10 @@ import (
 // ErrSegmentNotFound is returned if the segment could not be found.
 var ErrSegmentNotFound = errors.New("segment not found")
 
+// ErrIncorrectOffset is returned if the offset is incorrect. This is used in case Optimistic
+// Concurrency Control is activated
+var ErrIncorrectOffset = errors.New("Incorrect offset")
```

Comment:
```suggestion
var ErrIncorrectOffset = errors.New("incorrect offset")
```

Errors mustn't start with an uppercase letter.